src/src/CustomBuild/define_plugin_sets.h

```diff
@@ -1463,6 +1463,12 @@ To create/register a plugin, you have to :
     #ifndef NOTIFIER_SET_NONE
       #define NOTIFIER_SET_NONE
     #endif
+    #ifdef USES_N001
+      #undef USES_N001   // Email
+    #endif
+    #ifdef USES_N002
+      #define USES_N002   // Buzzer
```

Comment:
Shouldn't this be `#undef` ? (and the same for the USES_N002 below)

Reply:
Yes, it should. Fixed the typo 😊